### DISCUSSION TITLE
Mountain Duck on Sonoma with B2: Server connections interrupted

### DISCUSSION THREAD

**Discussion by GregWellmanJr** [iterate-ch/cyberduck#15327] [Q&A] [answered]
I am getting "Server connections interrupted" error messages from MacOS Sonoma 14.0 w/ Mountain Duck 4.15.0 whenever I try to upload large-ish files (147 MB) to Backblaze B2. Same situation whether connected via Smart Synchronization or Online mode. Downloading files doesn't seem to be an issue.

**Comment by tatze96** [comment]
Please send us a [debug log](https://docs.cyberduck.io/mountainduck/support/#debug-log) right after the message appears again within the *Online* mode to support@mountainduck.io for further investigation. We'll need the `mountainduck.log` file and any available `mountainduck-*.log.zip` files. Ensure to add the ticket reference GH-15327 so we can match the tickets quickly.

Regarding the message in general:
The server connection interrupted notification is a known issue. Finder closes the connection because Mountain Duck hasn't answered fast enough on the request of Finder. Although Finder indicates that the connection is interrupted, it usually isn't, as Mountain Duck is still connected to the server.

This is an issue within the operating system that can occur to any network drive.

The mounted drive will stay connected if you click on "Ignore" to dismiss the notification or wait until it disappears by itself.

**Comment by dkocher** [comment, marked as answer]
Discussed in #15735. We had another fix in version [4.17.0](https://mountainduck.io/changelog/) that should ensure the _Server connections interrupted_ prompt is not shown in Finder. Let us know if you can still reproduce and if there is a pattern.
